controller/tests/server/test_route_processing.py

```diff
@@ -1382,6 +1357,28 @@ def test_after_request__redacts_recording_folder_path_from_get_recordings_log_me
     }
 
 
+@pytest.mark.parametrize(
+    "mocked_request_method,expected_status_code,is_expecting_logs", [("GET", 200, False), ("PUT", 404, True)]
```

Comment:
This technically works, but /system_status doesn't allow `PUT` so like I mentioned before we should either use mocking or set up `shared_values_dict` to get /system_status to return something other than 200